### DISCUSSION TITLE
How to get all Archived Threads in given channel?

### DISCUSSION THREAD

**Discussion by blockchain0026** [discord-net/Discord.Net#2799] [Q&A]
I've did a lot of searching on the internet for querying archived threads using discord.net but found not thing, I know there is SocketTextChannel.Threads which returns a ReadOnlyCollection of threads, but it only returns what have cached, not all the threads.

**Comment by Sauceke** [comment]
Had this exact issue recently, [this ](https://github.com/ManlyMarco/CHEF/blob/b7bffa53db16e16526c0726219c9669ec69b914c/CHEF/Components/ModForumUpdater.cs#L146-L161C10)is how I solved it.
If you only want the archived threads, start with an empty list instead of `channel.GetActiveThreadsAsync()`.
